modules/ModuleMenus/src/client/ModuleMenus.coffee

```diff
@@ -88,9 +88,48 @@ class ModuleMenusController extends Backbone.View
 		if window.AppLaunchParams.deployMode?
 			unless window.AppLaunchParams.deployMode.toUpperCase() =="PROD"
 				@$('.bv_deployMode h1').html(_.escape(window.AppLaunchParams.deployMode.toUpperCase()))
-
+		@heartbeat()
 		@
 
+	heartbeat: =>
+		## Fetch /api/about endpoint, if 401, then notify user they have been logged out
+		## If another error then try something else
+		setInterval =>
+				originalStatus = @status
+				try
+					response = await fetch("/api/about", method: 'GET')
+					@status = response.status
+					@statusText = response.statusText
+				catch error
+					@status = "error"
+					@statusText = "Response error"
+				
+
+				# Check if the heartbeat notification exists and is active/visible
+				active = @heartbeatToast? && @heartbeatToast.isActive()
+
+				# If we aren't showing the heartbeat notification or the status has changed, then update/show the heartbeat notification
+				if !active || originalStatus != @status
+					if @heartbeatToast?
+						@heartbeatToast.closeToast()
+						@heartbeatToast.remove()
+					
+					if @status == 401
+						@heartbeatToast = new ACASToast
+							type: "error"
+							title: "Logged out"
+							text: "You have been logged out. Please login again."
```

Comment:
Can we indicate what the user's supposed to do? Reload the page, open a new tab? Show a link to the login page?
As a user I'd be unclear on how to "Please login again" if I was not being presented with the login page.

Reply:
Thanks, I added a feature to the toast notification widget where you can pass an "onClose" function.  I added a page reload to the heart beat notification for logged out such that when the user initiates a close by clicking the X, the page reloads (this is what LD does, albeit with a different notification than the bacon bar).  I also changed the text of the notification to match LDs "You have been logged out. You must log back in to continue."